psi4/src/psi4/detci/h0block.cc

```diff
@@ -358,11 +363,13 @@ void CIWavefunction::H0block_xy(double *x, double *y, double E)
       outfile->Printf("H0block_->s0bp[%d] = %lf\n",i,H0block_->s0bp[i]);
   */
 
-   dot_arr(H0block_->c0b, H0block_->c0bp, H0block_->size, &tx);
+   //dot_arr(H0block_->c0b, H0block_->c0bp, H0block_->size, &tx);
+   tx = C_DDOT(H0block_->size,  H0block_->c0bp, 1, H0block_->c0b, 1);
    *x += tx;
-   dot_arr(H0block_->s0b, H0block_->c0bp, H0block_->size, &ty);
+   //dot_arr(H0block_->s0b, H0block_->c0bp, H0block_->size, &ty);
+   ty = C_DDOT(H0block_->size,  H0block_->c0bp, 1, H0block_->s0b, 1);
```

Comment:
`DDOT` should be agnostic to input array order. If you see this in GEMM however...